### DISCUSSION TITLE
How I've set up my new Windows development environment in 2022

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#265] [Announcements]
# How I've set up my new Windows development environment in 2022

Configuring a new Windows machine with ease for full-stack .NET and Angular development

https://timdeschryver.dev/blog/how-i-have-set-up-my-new-windows-development-environment-in-2022

**Comment by jbakeri4** [comment]
The article may be a little out of date, but the tips aren't! Thank you.

**Reply by timdeschryver** [reply]
Enjoy! 🕺

**Comment by profjobell** [comment]
Thank you very much for this Tim. I'm relatively new to Development in IT field, and your helpful posting directly answered may question on setting up the optimal environment on my Windows PC. ( I work a lot in Mac on Documentation and Media Creation. )  
Thanks again. :)
Prof Joe Bell

**Reply by timdeschryver** [reply]
I'm glad that you found this post useful @profjobell !
